apps/web/src/components/PasswordInput/PasswordInput.tsx

```diff
@@ -35,19 +37,45 @@ export const PasswordInput = <T extends FieldValues, U extends Path<T>>({
   placeholder = "Enter your password",
   required = "Password is required",
   minLength = MIN_LENGTH,
+  isStrengthCheckEnabled = false,
   validate,
   ...rest
 }: PasswordInputProps<T, U>) => {
+  const form = useFormContext<T>();
   const {
     register,
     formState: { errors },
-  } = useFormContext<T>();
+  } = form;
   const [showPassword, setShowPassword] = useState<boolean>(false);
+  const { validatePasswordStrength, PasswordStrengthBar } = usePasswordValidation({
+    inputName,
+    minLength,
```

Comment:
Why MIN_LENGTH is defined here and not in usePasswordValidation? 
Could it have values other than 12?

Reply:
Is it to use it here for the error message? 
Maybe instead we can keep everything related to the validation in `usePasswordValidation`, and export the minLength from there?